tests/k8st/tests/test_bgp_advert.py

```diff
@@ -160,12 +160,6 @@ def setUp(self):
 
         start_external_node_with_bgp("kube-node-extra", bird_conf)
 
-        # set CALICO_ADVERTISE_CLUSTER_IPS=10.96.0.0/12
```

Comment:
Do we not need to replace this with a call to set up the BGPConfiguration?

Reply:
I moved this to the start of each of the non-rr tests. I considered making a function out of this to template the clusterIP and externalIP fields but not sure if it's worth it or not.

Reply:
I see you've done it in the individual tests. Makes sense, since we have one test that doesn't use clusterIPs.